src/client/datascience/jupyter/kernels/kernelSelections.ts

```diff
@@ -307,6 +47,15 @@ export class KernelSelectionProvider {
         LiveKernelConnectionMetadata | KernelSpecConnectionMetadata
     >[] = [];
     private _listChanged = new EventEmitter<Resource>();
+    /**
+     * List of ids of kernels that should be hidden from the kernel picker.
+     *
+     * @private
+     * @type {new Set<string>}
+     * @memberof KernelSelector
```

Comment:
Auto doc comments are now out of date, and not useful anyways. I'd just keep the comment.